scst-store/amr-graphql-query.hbs.md

```diff
@@ -0,0 +1,255 @@
+# AMR Graphql Querying
+
+This page contains information on how to connect to the graphql playground, and how to query each supported data model.
+See [AMR Data Models](amr-data-model.hbs.md) for more information on the AMR data models.
+
+## Connecting to Graphql Playground
+
+Having ingress enabled is the recommended configuration by VMWare. The SCST-Store and AMR share the same ingress configuration, so setting the Store ingress domain will also set the same for AMR.
+To connect to the AMR Graphql playground when ingress is enabled, go to `https://artifact-metadata-repository.<ingress-domain>/play`.
+On this page users will be able to write and execute their own graphql queries to fetch data from the AMR.
+
+## Querying for Locations
+
+This section provides details on the query parameters accepted for `Location`, and the list of attributes available for `Location`.
+
+### <a id='location-query-params'></a> Location Query Parameters
+
+Below are the supported parameters that users can specify when querying for `Location`, they are all optional.
+- `query`: expects a query object that specifies additional query parameters, supported parameters in this query object are listed below
+  - `alias`: name of the location to query for, `String` value. By default `alias` is set to be the same as the `reference`, but it is configurable by the user, see [here](link)
+    
+    Example:
+    ```
+    locations(query:{alias: "test-cluster"})
+    ```
+    
+  - `reference`: reference of the location to query for, which is the `kube-system` namespace UID, `String` value.
+  
+    Example:
+    ```
+    locations(query:{reference: "f4f63c6a10ed7cfb06dfb03c2b2d6a9d5bbe95931c71f8cb346fd2284b1e5d82"})
+    ```
+    
+  - `labels`: query for locations with specified labels, expects a list of labels. 
```

Comment:
I think since `query` is in the actual json body, we should use another word or rephrase such that we don't have to overload the term `query` in this doc and hopefully prevent confusion.